src/middlewared/middlewared/plugins/catalogs_linux/items.py

```diff
@@ -52,15 +52,15 @@ def items(self, label, options):
 
     @private
     def get_trains(self, location, options=None):
-        # We make sure we do not dive into library folder and not consider it a train
-        # This allows us to use this folder for placing helm library charts
+        # We make sure we do not dive into library and docs folders and not consider those a train
+        # This allows us to use these folders for placing helm library charts and docs respectively
         trains = {'charts': {}, 'test': {}}
         options = options or {}
         unhealthy_apps = set()
         for train in os.listdir(location):
             if (
                 not os.path.isdir(os.path.join(location, train)) or train.startswith('.') or train == 'library' or
```

Comment:
Let's please do `train in ('library', 'docs')`, easier to extend if needed moving on

Reply:
fixed and moved to the second row to make for easier expansion in the future.